Remove the computer image

x5hwuk7nc6jsudcc opened the pull request.

davidgatti reviewed (commented):
The device screen is still in place, and white as seen in this screenshot bellow. Please disable the color or remove the device screen altogether.

![Screen Shot 2021-06-02 at 7 56 40 AM](https://user-images.githubusercontent.com/411114/120431081-49685c00-c378-11eb-9e68-ac16bd5ddcfe.png)

x5hwuk7nc6jsudcc commented:
I've rewritten the 'feature' utility and removed the "device" overlay entirely. My instinct was to keep the original theme functionality, but on reflection I agree that if it's not being used, it's better to just remove it rather than work around it in this case. Please re-review PR @davidgatti .
